docs_user/modules/proc_stopping-openstack-services.adoc

```diff
@@ -2,30 +2,22 @@
 
 = Stopping {rhos_prev_long} services
 
-Before you start the adoption, you must stop the {rhos_prev_long} ({OpenStackShort}) services.
+Before you start the {rhos_long} adoption, you must stop the {rhos_prev_long} ({OpenStackShort}) services to avoid inconsistencies in the data that you migrate for the data plane adoption. Inconsistencies are caused by resource changes after the database is copied to the new deployment.
 
-This is an important step to avoid inconsistencies in the data migrated for the data-plane adoption procedure caused by resource changes after the DB has been copied to the new deployment.
+You should not stop the infrastructure management services yet, such as:
 
-Some services are easy to stop because they only perform short asynchronous operations, but other services are a bit more complex to gracefully stop because they perform synchronous or long running operations that you might want to complete instead of aborting them.
-
-Since gracefully stopping all services is non-trivial and beyond the scope of this guide, the following procedure uses the force method and presents
-recommendations on how to check some things in the services.
-
-Note that you should not stop the infrastructure management services yet, such as:
-
-* database
+* Database
 * RabbitMQ
 * HAProxy Load Balancer
-* ceph-nfs
+* Ceph-nfs
 * Compute service
-* containerized modular libvirt daemons
-* {object_storage_first_ref} backend services
+* Containerized modular libvirt daemons
+* {object_storage_first_ref} back-end services
 
 .Prerequisites
 
 * Confirm that there no long-running operations that require the services you plan to stop.
-* Ensure that there are no ongoing instance live migrations, volume migrations (online or offline), volume creation, backup restore, attaching, detaching,
-and so on.
+* Ensure that there are no ongoing instance live migrations, volume migrations (online or offline), volume creation, backup and restore, attaching, detaching, and so on:
```

Comment:
Maybe this line could be combined with the previous one like: " Ensure that there are no long-running tasks that require the services that you plan to stop, such as instance live migrations, volume migrations, volume creation, backup and restore, attaching, detaching, and other operations."
?